src/Features/CSharp/Portable/CodeRefactorings/MoveStaticMembers/CSharpMoveStaticMembersRefactoringProvider.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Composition;
+using System.Diagnostics.CodeAnalysis;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeRefactorings;
+using Microsoft.CodeAnalysis.Host.Mef;
+using Microsoft.CodeAnalysis.MoveStaticMembers;
+
+namespace Microsoft.CodeAnalysis.CSharp.CodeRefactorings.MoveStaticMembers
+{
+    [ExportCodeRefactoringProvider(LanguageNames.CSharp, Name = PredefinedCodeRefactoringProviderNames.MoveStaticMembers), Shared]
+    internal class CSharpMoveStaticMembersRefactoringProvider : AbstractMoveStaticMembersRefactoringProvider
+    {
+        /// <summary>
+        /// Test purpose only.
+        /// </summary>
+        [SuppressMessage("RoslynDiagnosticsReliability", "RS0034:Exported parts should have [ImportingConstructor]", Justification = "Used incorrectly by tests")]
+        public CSharpMoveStaticMembersRefactoringProvider(IMoveStaticMembersOptionsService? service) : base(service)
+        {
+        }
```

Comment:
This constructor is incorrect based on https://github.com/dotnet/roslyn-analyzers/issues/5358:

> MEF-exported types should have at most one public constructor, which should either have no parameters or be marked [ImportingConstructor]

FYI @sharwell :)

Reply:
I'll be meeting with him to change the test framework, it will be addressed in a follow-up PR 👍